src/ppx/application.cpp

```diff
@@ -800,13 +800,18 @@ void Application::SaveMetricsReportToDisk()
 void Application::InitStandardKnobs()
 {
     // Flag names in alphabetical order
-    std::vector<std::string> defaultAssetsPaths = {};
+    std::vector<std::string> defaultEmptyList = {};
     mStandardOpts.pAssetsPaths =
-        mKnobManager.CreateKnob<KnobFlag<std::vector<std::string>>>("assets-path", defaultAssetsPaths);
+        mKnobManager.CreateKnob<KnobFlag<std::vector<std::string>>>("assets-path", defaultEmptyList);
     mStandardOpts.pAssetsPaths->SetFlagDescription(
         "Add a path in front of the assets search path list.");
     mStandardOpts.pAssetsPaths->SetFlagParameters("<path>");
 
+    mStandardOpts.pConfigJsonPath = mKnobManager.CreateKnob<KnobFlag<std::vector<std::string>>>("config-json-path", defaultEmptyList);
+    mStandardOpts.pAssetsPaths->SetFlagDescription(
+        "Additional commandline flags specified in a JSON file.");
```

Comment:
You should describe the complete behavior:
`-flag=1 -json=toto.json -json=tata.json`
-> tata.json values override values from toto.json

`-flag=1 -json=toto.json -flag=2`
-> toto.json overrides any values, even if `flag` is specified after.